addons/binding/org.openhab.binding.helios/src/main/java/org/openhab/binding/helios/ws/rest/RESTSwitch.java

```diff
@@ -0,0 +1,12 @@
+package org.openhab.binding.helios.ws.rest;
+
+public class RESTSwitch {
```

Comment:
Please add a JavaDoc with `@author` tag containing a real name of the author. See [Code Style](http://docs.openhab.org/developers/development/guidelines.html#a-code-style).